lib/advanced_messenger_issues_controller_patch.rb

```diff
@@ -0,0 +1,57 @@
+# This module could not be named IssuesControllerPatch
+# because another module with same name exists in our other plugin: redmine_crispico_plugin, 
+# and when requesting IssuesControllerPatch from redmine_crispico_plugin, it was resolved to IssuesControllerPatch from redmine_advanced_messenger
+# That's why we prefixed it with the plugin name  
+module AdvancedMessengerIssuesControllerPatch
+    include ActionView::Helpers::JavaScriptHelper
+    def self.included(base)
+      base.send(:include, InstanceMethods)
+      base.send(:after_action, :expand_collapse_notes, only: [:show])
+    end
+    
+    module InstanceMethods
+      def expand_collapse_notes
+        expand_collapse_notes_or_messages(@journals, lambda {|journal| journal.notes[0, 130]}, lambda {|journal| journal.notes.nil?},
+              # This regular expresion catches:
+              #   <div id="journal-42-notes" class="wiki">
+              #     <p>my note content</p>
+              #   </div> 
+              # and creates capture groups, in order to replace and obtain:
+              #   <div id="journal-42-notes" class="wiki">
+              #     <div class="message-content">
+              #       <p>my note content</p>
+              #     </div>
+              #     <div class="message-preview">
+              #     </div>
+              #   </div>
+              lambda {|journal_id| '(<div id="journal-' + journal_id + '-notes" class="wiki">)((?:.|\n)*?)(<\/div>)'},
```

Comment:
Yes, you are right, but there is no need for such a match because the div that contains the test (wiki) has already the id on it (the id is doubled on 2 divs):
![image](https://github.com/famiprog/redmine_advanced_messenger/assets/4996054/c182e435-1139-47c9-8b3f-d4bbabe7aba4)